scripts/fires/wfigs_data.py

```diff
@@ -0,0 +1,278 @@
+                geoIds = ll2p.resolve(latitude, longitude)
+                for geoId in geoIds:
+                    if geoId not in ('northamerica', 'country/CAN',
+                                     'country/MEX'):
+                        location += 'dcid:%s, ' % geoId
+                if 'northamerica' in geoIds:
+                    return_val = (location + ('[LatLong %s %s]' %
+                                              (latitude, longitude)))
+                    if latitude in _CACHE:
+                        _lat_dict = CACHE[latitude_str]
+                        _lat_dict[longitude_str] = return_val
+                    else:
+                        _CACHE[latitude_str] = {longitude_str: return_val}
+                    return return_val
+            if not row.POOFips or pd.isna(row.POOFips):
+                location = mapping.POOSTATE_GEOID_MAP[row.POOState]
+                return location
+            else:
+                return None
+        except Exception as e:
+            logging.debug("Failed resolution for ({0},{1} for fire {2})".format(
+                latitude, longitude, row.UniqueFireIdentifier))
+            return ''
+
+    # Get FIPS codes associated with each incident.
+    def get_fips(row):
+        if row.POOFips and not pd.isna(row.POOFips):
+            fips_str = (row.POOFips.zfill(5))
+            return "dcid:%s, dcid:%s, dcid:%s" % (
+                "geoId/" + fips_str[:2], "geoId/" + fips_str, "country/USA")
+        else:
+            return None
+
+    # Write expected loss in an ingest-able format.
+    def get_cost(cost_val):
+        if cost_val and not np.isnan(cost_val):
+            return _EXPECTED_LOSS_MAP.format_map({'costs': cost_val})
+        else:
+            return None
+
+    # Write fire area in an ingest-able format.
+    def get_area(area, label, mapping):
+        if area and not np.isnan(area):
+            return mapping.format_map({label: area})
+        else:
+            return None
+
+    # Get cause-dcid for the fire cause.
+    def get_cause(cause_str):
+        if cause_str:
+            cause = cause_str.replace(" ", "").replace("/", "Or")
+            if cause in mapping.CAUSE_DCID_MAP:
+                cause = mapping.CAUSE_DCID_MAP[cause]
+            return cause
+        else:
+            return None
+
+    # get personnel count for the incident.
+    def get_personnel(personnel_val):
+        if personnel_val and not np.isnan(personnel_val):
+            return personnel_val
+        else:
+            return None
+
+    # If a parent fire exists, associate it with given fire.
+    def get_parent_fire(row):
+        if not (np.isnan(row.IsCpxChild)) and row.IsCpxChild == 1 and row.CpxID:
+            # remove curly brackets from front and end of string.
+            return '{prefix}{id}'.format(prefix=_FIRE_DCID_PREFIX,
+                                         id=row.CpxID[1:-1].lower())
```

Comment:
please move this above so it's right below the comment on 213 (assign it to a local var for readability)

Reply:
Done